ksqldb-engine/src/main/java/io/confluent/ksql/engine/rewrite/AstSanitizer.java

```diff
@@ -221,12 +221,11 @@ public static Statement sanitize(final Statement node, final MetaStore metaStore
       ctx.process(expression.getBody());
       ctx.getContext().removeLambdaArgs();
       ctx.getContext().addLambdaArgs(previousLambdaArgs);
-
       return visitExpression(expression, ctx);
     }
   }
 
-  private static class SanitizerContext {
+  public static class SanitizerContext {
```

Comment:
nit: sorry, another random tidbit leftover from my initial design, we should remove this

Reply:
IDE is complaining when I try to change it back to public